src/browser/atom-portable.coffee

```diff
@@ -4,31 +4,34 @@ ipc = require 'ipc'
 
 module.exports =
 class AtomPortable
-  @getPortableAtomHomePath: ->
+  @getPortableAtomHomePath: (platform) ->
+    # Mac has a deeper path for the executable in the application package, so need a different
```

Comment:
I'm actually hoping to go through the inconsistencies over the weekend in an attempt to resolve them. So not a major thing if you don't fix this one now, but I'll have to fix it eventually :)